### DISCUSSION TITLE
QField Cloud not connecting PostGIS (Postgres Server)

### DISCUSSION THREAD

**Discussion by Greymorn** [opengisch/QField#2996] [Q&A]
I am having problem getting QField Cloud to connect to my PostGIS database. I have had our system admin add the IP to our fire wall to let it through and I have the information both stored in the project and the Secrets Page and I keep getting the following errors.

![Untitled](https://user-images.githubusercontent.com/107422507/174664041-5475a77a-4091-4939-9e63-e5c77293e158.png)

I have tried connecting to the database from an outside connection and it work no problem with PGAdmin and QGIS for desktop. I don't know if I am missing an option or if there is something else that I need to open up. I have really scoured the net but have found nothing.

Any help would be awesome

**Comment by suricactus** [comment]
Hi @Greymorn . Are you sure you configured your project correctly so it uses a the pgservice configuration? Can you upload your project again and share it's name, so we can investigate deeper?

**Reply by Greymorn** [reply]
Hey @suricactus kr_gsca/Q_Field_Testing would be the project name. Do I have to make the project public for you to find it? let me know if you need anything else, and thank for looking into this.

**Reply by suricactus** [reply]
No, all is good, elevated users have extra permissions to see private projects for support.

**Reply by Greymorn** [reply]
Hey @suricactus I have changed the project name as I thought I would try something. It is now kr_gsca/Testing_4.
